### ISSUE TITLE
bug in scaffoldWidget

### ISSUE THREAD

**Issue by timelyportfolio** [ramnathv/htmlwidgets#210]
I now get an error with `scaffoldWidget`.

```
> htmlwidgets::scaffoldWidget("mywidget")
Created boilerplate for widget constructor R/mywidget.R
Error in editor(file = file, title = title) : 
  argument "name" is missing, with no default
```

Nothing changed in the code for a while, so I will explore what is happening.

**Comment by ramnathv** [comment]
I noticed some changes in how `file.edit` works in `RStudio` now. Not sure if this is connected to that. I am unable to reproduce this issue however. What version of `RStudio` are you on?

**Comment by daattali** [comment]
I'm getting the same issue right now.  RStudio Version 0.99.1251, htmlwidgets v0.6

**Comment by jjallaire** [comment]
PR here: https://github.com/ramnathv/htmlwidgets/pull/214

**Comment by ramnathv** [comment]
Closing this as it has been fixed by #214.
